tests/Unit/Analyzer/FileVisitorTest.php

```diff
@@ -730,4 +730,34 @@ public function getBookList(): QueryBuilder;
 
         $this->assertCount(1, $violations);
     }
+
+    public function test_it_skip_custom_tags_in_docblocks_the_option_parse_doc_block_custom_tags_is_false(): void
+    {
+        $code = <<< 'EOF'
+<?php
+namespace MyProject\AppBundle\Application;
+use Symfony\Component\Validator\Constraints as Assert;
+class ApplicationLevelDto
+{
+/**
+* @Assert\NotBlank
+*/
+     public $foo;
+
+}
+EOF;
+
+        /** @var FileParser $fp */
+        $fp = FileParserFactory::createFileParser(TargetPhpVersion::create('8.1'), false);
+        $fp->parse($code, 'relativePathName');
+
+        $cd = $fp->getClassDescriptions();
+
+        $violations = new Violations();
+
+        $notHaveDependencyOutsideNamespace = new DependsOnlyOnTheseNamespaces('MyProject\AppBundle\Application');
```

Comment:
Can you please change the var name with the correct one?

Reply:
@AlessandroMinoccheri done!